steps/step1.py

```diff
@@ -65,55 +60,69 @@ def filter_by_rules(df) -> pd.DataFrame:
     dropped_months = 0
 
     # Set path for drop rules file
-    subdirectory_path = "parameters/drop_rules.csv"
-    drop_rules_path = os.path.join(parent_dir, subdirectory_path)
+    parent_directory = os.path.dirname(os.getcwd())
+    subdirectory_path = "GISTEMP_rewrite/parameters/drop_rules.csv"
```

Comment:
A few things: 1) Just a note that the subdirectory_path will need to be changed in production code to whatever the new dir is (GISTEMP_v5?)
2) But now that I am looking at it, why is "GISTEMP_rewrite" specified? Shouldn't that be the current working directory? and so the subdirectory_path would be "parameters/drop_rules.csv" as it was before. I am sure I am missing something here, since the process works.
3) We may want to include a note in the readme that users should run from a specific directory OR we could update the code to ignore the parent directory and just use a relative path (probably the best bet). So they can export the parent dir path and run from wherever